released/packages/coq-poltac/coq-poltac.0.8.12/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+maintainer: "thery@sophia.inria.fr"
+homepage: "https://github.com/thery/PolTac"
+bug-reports: "https://github.com/thery/PolTac"
+dev-repo: "git://github.com/thery/PolTac"
+license: "LGPL"
```

Comment:
@thery could you be more specific about the license? I'm guessing this is either `LGPL-2.1-only` or `LGPL-2.1-or-later`.